### ISSUE TITLE
nfs service name broken in centos < 7

### ISSUE THREAD

**Issue by koglinjg** [example42/puppet-nfs#6]
I see issue #3 is to fix the service name for centos 7, however it is now broken for < 7. I would recommend a structure like https://github.com/puppetlabs/puppetlabs-java/blob/1.3.0/manifests/params.pp so you can apply version specific service and package names. I wouldn't mind making a pull request. Just let me know that is a structure that looks ok to you.

My current workaround while using your module is to use
```
class { '::nfs':
  ...
  service => 'nfs'
  ...
}
```

There being an easy workaround I would consider this a minor bug.

**Comment by alvagante** [comment]
Yes, that approach would be ok. A PR would be very welcomed, thank you
